src/Search/ElasticaToModelTransformerCollection.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace App\Search;
```

Comment:
Because we didn't really write this file, I do want move this to up the `Transformer` namespace.